Mathlib/Tactic/ToAdditive.lean

```diff
@@ -106,7 +106,7 @@ Examples:
 * `@Mul.mul Nat n m` (i.e. `(n * m : Nat)`) will not change to `+`, since its
   first argument is `Nat`, an identifier not applied to any arguments.
 * `@Mul.mul (α × β) x y` will change to `+`. It's first argument contains only the identifier
-  `prod`, but this is applied to arguments, `α` and `β`.
+  `Prod`, but this is applied to arguments, `α` and `β`.
```

Comment:
I'm not sure if this change is desirable. If not, here's a revert:
```suggestion
  `prod`, but this is applied to arguments, `α` and `β`.

Reply:
This one is good